cou/utils/juju_utils.py

```diff
@@ -192,7 +197,9 @@ async def _check_action_error(action_obj, model, raise_on_failure):
         raise ActionFailed(action_obj, output=output)
 
 
-async def async_run_on_unit(unit_name, command, model_name=None, timeout=None):
+async def async_run_on_unit(
+    unit_name: str, command: str, model_name: Optional[str] = None, timeout: Optional[int] = None
+) -> Dict[str, str]:
     """Juju run on unit.
 
     :param model_name: Name of model unit is in
```

Comment:
Preferably change the sequence of the parameters to match the order

change `:type model_name: str` to `:type model_name: Optional[str]`
change `:type unit: str` to `:type unit_name: str`
change `:type timeout: int` to `:type timeout: Optional[int]`
change `:rtype: dict` to `:rtype: Dict[str, str]`

Reply:
ok